hw/riscv/sifive_u.c

```diff
@@ -295,40 +297,15 @@ static void riscv_sifive_u_init(MachineState *machine)
         SIFIVE_U_PLIC_CONTEXT_BASE,
         SIFIVE_U_PLIC_CONTEXT_STRIDE,
         memmap[SIFIVE_U_PLIC].size);
-    sifive_uart_create(sys_memory, memmap[SIFIVE_U_UART0].base,
+    sifive_uart_create(system_memory, memmap[SIFIVE_U_UART0].base,
         serial_hds[0], SIFIVE_PLIC(s->plic)->irqs[SIFIVE_U_UART0_IRQ]);
-    /* sifive_uart_create(sys_memory, memmap[SIFIVE_U_UART1].base,
+    /* sifive_uart_create(system_memory, memmap[SIFIVE_U_UART1].base,
```

Comment:
FWIW, there are two UARTs on the U540-C000

Reply:
The second UART is disabled until we can figure out how to plug them both in without getting "make check" failures. That's why i've left the code commented. I should probably add a further comment. The serial frontend/backend has changed a lot since we rebased to upstream QEMU so it may be possible to fix the issue. We also have to figure out how to plug hardware from a config file or the command line given 'sifive_u' and 'sifive_e' are going to be somewhat generic.